### ISSUE TITLE
YT Sync

### ISSUE THREAD

**Issue by tibfox** [dtube/DTubeGo#203]
1. verification that the user is the owner of the youtube channel via code in bio
2. actual sync

**Comment by tibfox** [comment]
The sync needs a proposal for the dao and some extensive time to code / test

**Comment by brishtiteveja** [comment]
yeah, this is the most interesting and important feature, in my opinion.
